app/rev2/prelaunch/test_prelaunch.c

```
@@ -105,8 +111,8 @@ void test_check_config_validity() {
 }
 
 void test_preset_cmd_execute() {
-	HFLASH_BUFFER* flash_handle;
-	uint32_t* flash_address;
+	HFLASH_BUFFER flash_handle;
+	uint32_t flash_address;
```

Clang (14.0.6)

[warning] variable 'flash_address' is not initialized

(cppcoreguidelines-init-variables)